app/backend/src/couchers/interceptors.py

```diff
@@ -155,7 +156,12 @@ def tracing_function(request, context):
                 user_id = getattr(context, "user_id", None)
                 self._store_log(method, code, duration, user_id, request, None, traceback)
                 self._observe_in_histogram(method, code or "", type(e).__name__, duration)
-                sentry_sdk.capture_exception(e)
+
+                if code not in (grpc.StatusCode.NOT_FOUND, grpc.StatusCode.UNAUTHENTICATED):
```

Comment:
I think we should only send exceptions that were not raised by us in `context.abort`. Those will have `code` be `None` in this bit here. So you could change this line to `if not code`